tinygrad/runtime/ops_torch.py

```diff
@@ -9,8 +9,10 @@
 device = torch.device("cuda:0" if torch.cuda.is_available() else ("mps" if getenv("MPS", 0) else "cpu"))
 type_map = {torch.float64: dtypes.float64, torch.float16: dtypes.float16, torch.float32: dtypes.float32,
             torch.int8: dtypes.int8, torch.int32: dtypes.int32, torch.int64: dtypes.int64,
-            torch.uint8: dtypes.uint8, torch.bool: dtypes.bool, torch.int16: dtypes.int16}
-inverse_type_map = {v:k for k,v in type_map.items()}
+            torch.uint8: dtypes.uint8, torch.bool: dtypes.bool, torch.int16: dtypes.int16,
+            torch.bfloat16: dtypes.bfloat16}
+inverse_type_map = dict([(v, k) for k,v in type_map.items()] + [(dtypes.ushort, torch.int16), (dtypes.uint, torch.int32)])
```

Comment:
what happens for stuff outside of int16 or int32 range?

Reply:
Clipping would be my guess. In reality those are quantized models already, so probably not a big deal, the output of coder seems coherent still.